### ISSUE TITLE
alpha1: single line blocks not allowed anymore?

### ISSUE THREAD

**Issue by mitchellh** [hashicorp/terraform#19153]
I'm not sure if this change is intentional or not but it seems not ideal. When you have a lot of little variables it was nice to create a bunch of one-liners and now it seems I have to break it up. 

This is particularly sharp in the project I'm trying this in, because there are ~50 variables so it's a lot of work to fix this up too. 😄 

### Terraform Version
```
Terraform v0.12.0-alpha1
```

### Terraform Configuration Files
```hcl
variable "region" { default = "us-east-1" }
```

### Expected Behavior
Should work

### Actual Behavior
```
Error: Missing newline after argument

  on vars.tf line 22:
  22: variable "region" {default = "us-east-1"}

An argument definition must end with a newline.
```

### Steps to Reproduce
Create a file with the variable and run an `init`.

### Additional Information
Note that if the variable is EMPTY it works:

```hcl
variable "region" {}
```

**Comment by mitchellh** [comment]
Amusingly, if I run `fmt` with Terraform 0.11, it actually _fixes_ all of this to be multi-line and spread out.

I thought we used to format single-liners but I guess we got rid of that at some point. That's all okay, but is broken in 0.12 (and `fmt` doesn't work on it either)!

**Comment by apparentlymart** [comment]
Fixed as part of #19664
